### PR TITLE
[FABP-269] Modify CONTRIBUTING.md to be better understand

### PR DESCRIPTION
**Pull request by Lucas-Wye**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The project employs [Github](https://github.com/hyperledger/fabric-sdk-py) as th
 *Before committing code, please go to [Jira](https://jira.hyperledger.org/secure/RapidBoard.jspa?rapidView=85) to create a new task or check if there's related existing one, then assign yourself as the assignee. Notice each task will get a Jira number like [FABP-3082](https://jira.hyperledger.org/browse/FABP-3082).
 
 
-* Clone the project into your working directory with your Github ID.
+* Clone project `cello` into your working directory with your Github ID.
```
Comment (yeasy) [inline CONTRIBUTING.md]: Seems line 40-48 should be removed, as duplicated with line 56-63

Comment (Lucas-Wye) [reply CONTRIBUTING.md]: OK, that is quite confused when I read this doc. Actually line 40-48 aren't associated with this project.

Comment (yeasy) [reply CONTRIBUTING.md]: Yes, please help revise the pr to remove the duplicated lines.

Comment (Lucas-Wye) [reply CONTRIBUTING.md]: Done.

**Pull request merged**